libs/config/src/lib/database.config.spec.ts

```
@@ -0,0 +1,32 @@
+  it('should return the correct database host and port from environment variables', () => {
+    process.env['DATABASE_HOST'] = 'localhost';
+    process.env['DATABASE_PORT'] = '3306';
+
+    const config = databaseConfig();
+    expect(config.host).toBe('localhost');
+    expect(config.port).toBe('3306');
+  });
+
+  it('should return the default port if DATABASE_PORT is not set', () => {
+    process.env['DATABASE_HOST'] = 'localhost';
+    delete process.env['DATABASE_PORT'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)